packages/suite/src/services/suite/invityAPI.ts

```diff
@@ -418,7 +360,7 @@ class InvityAPI {
         try {
             const response: SellFiatTrade = await this.request(
                 this.SELL_FIAT_CONFIRM,
-                trade,
+                trade as any,
```

Comment:
We should not need `any` here (or anywhere else 😉)
```suggestion
                trade,
```